pages/common/ansible.md

```diff
@@ -0,0 +1,27 @@
+# ansible
+
+> Manage groups of computers remotely over SSH
+
+- Display a list of hosts belonging to a group:
+
+`ansible {{group}} --list-hosts`
+
+- Ping a group of hosts:
+
+`ansible {{group}} --module-name ping`
+
+- Display facts about a group of hosts:
+
+`ansible {{group}} --module-name setup`
```

Comment:
By using single-letter options, it will conflict with contribution guideline point 3. I just want to confirm that this is okay in this case? I have no personal preference for either style, as long as we're consistent about it.

My thoughts on this:

- I agree that we should mention modules in the description to provide a pathway for beginners to discover advanced topics, as well as to hint at the correct mental model
- I feel it's easier for the beginner user to skim quickly for what they're looking for if we start the sentences with the desired outcome as opposed to technical specifics. My intuition tells me that the user will be thinking in terms of results, e.g. "I want to run a command" or "I want to ping hosts", as opposed to "I want to invoke module X".

My preferred solution would be something in the vein of "Display facts about a group of hosts, using the setup module", though I wouldn't mind being overridden as long as consistency is maintained.